app/src/main/java/com/allan/boardbuddies/fragments/NotesFragment.java

```diff
@@ -0,0 +1,67 @@
+package com.allan.boardbuddies.fragments;
+
+import android.os.Bundle;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.allan.boardbuddies.Note;
+import com.allan.boardbuddies.NoteAdapter;
+import com.allan.boardbuddies.R;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+import com.google.android.material.snackbar.Snackbar;
+
+import java.util.ArrayList;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link NotesFragment newInstance} factory method to
+ * create an instance of this fragment.
+ *
+ */
+public class NotesFragment extends Fragment {
+    private ArrayList<Note> notes = new ArrayList<Note>();
+    private RecyclerView recyclerView;
+    private FloatingActionButton addNoteFab;
+    private NoteAdapter adapter;
+    public NotesFragment() {
+        // Required empty public constructor
+    }
```

Comment:
pretty sure you don't need this? it should automatically create default constructor

Reply:
docs say i should have it